contracts/colony/ColonyStorage.sol

```diff
@@ -237,21 +239,21 @@ contract ColonyStorage is CommonStorage, ColonyDataTypes, ColonyNetworkDataTypes
     _;
   }
 
-  modifier auth override {
-    require(isAuthorized(msg.sender, 1, msg.sig), "ds-auth-unauthorized");
+  modifier auth virtual override {
```

Comment:
Why?

Reply:
Bump

Reply:
I definitely did deal with this... I assume I lost it in a rebase somewhere.